File: lib/fizzbuzz.rb
def fizzbuzz(n)
  raise ArgumentError unless n.is_a?(Fixnum)
  str = ""
  str << "Fizz" if n % 3 == 0
  str << "Buzz" if n % 5 == 0
  str = "#{n}" if str.empty?
  str
end

File: lib/fizzbuzz.js
function Fizzbuzz() {
  var i = 0;
  while (i < 1000) {
    var s = '';
    if (i % 3 == 0) { s += 'Fizz'};
    if (i % 5 == 0) { s += 'Buzz'};
    if (s == '') { s = i };
    console.log(s);
    i += 1;
  }
}

Fizzbuzz()
